src/account/models.py

```diff
@@ -1,3 +1,64 @@
+from django.contrib.auth.base_user import AbstractBaseUser
+# from main.managers import CustomerManager
+# from main.utils.validators import first_name_validator
+from django.contrib.auth.models import PermissionsMixin
 from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from phonenumber_field.modelfields import PhoneNumberField
 
-# Create your models here.
+from account.managers import CustomerManager
+
+
+class Customer(AbstractBaseUser, PermissionsMixin):
+    first_name = models.CharField(_("first name"), max_length=150, blank=True)
+    last_name = models.CharField(_("last name"), max_length=150, blank=True)
+    email = models.EmailField(_("email address"), unique=True, error_messages="A user with that email already exists.")
+    avatar = models.ImageField(upload_to="customer/avatar_customer/", null=True, blank=True)
+    phone_number = PhoneNumberField(blank=True, null=True, help_text="+1234567890")
+    date_joined = models.DateTimeField(_("date joined"), default=timezone.now)
+    birth_date = models.DateField(_("birth_day"), blank=True, null=True)
+    city = models.CharField(_("city"), max_length=150, blank=True)
+    is_staff = models.BooleanField(
+        _("staff status"),
+        default=False,
+        help_text=_("Designates whether the user can log into this admin site."),
+    )
+    is_active = models.BooleanField(
+        _("active"),
+        default=True,
+        help_text=_(
```

Comment:
you have selling web site, maybe you need to add some additional fields like, address, and preferred currency.